docker/Dockerfile.static

```diff
@@ -8,6 +8,7 @@ RUN apk add --update \
   bash \
   bcc-dev \
   bcc-static \
+  benchmark-dev \
```

Comment:
Any reason not to add these to the other dockerfiles?

Reply:
Yeah, I will need to do that. This is the only one that is tested? Given the the purpose of the other files is to test the build, should I move those into CI?

Reply:
The other dockerfiles are only run on master. They tend to be flaky for reasons outside our control - we run against unstable tip. So didn't wanna bog down PRs with their flakiness.

Reply:
I've dropped this dependency completely.